src/main/java/com/ays/assignment/service/impl/handler/AssignmentStartHandler.java

```diff
@@ -0,0 +1,39 @@
+package com.ays.assignment.service.impl.handler;
+
+
+import com.ays.assignment.model.entity.AssignmentEntity;
+import com.ays.assignment.model.enums.AssignmentHandlerType;
+import com.ays.assignment.model.enums.AssignmentStatus;
+import com.ays.assignment.repository.AssignmentRepository;
+import com.ays.auth.model.AysIdentity;
+import com.ays.user.model.enums.UserSupportStatus;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class AssignmentStartHandler extends AssignmentAbstractHandler {
```

Comment:
Bu sınıfın dışarıdan çağrılmayacağını düşünüyorum, bu yüzden yapabiliyorsak `package-private` yapabiliriz.

Reply:
Test tarafında mock'layabilmek için ulaşmam gerekiyordu. Bu yüzden handler'ları public yaptım.

Reply:
Standart şartlarda test için buna ihtiyaç olmaması gerekiyor, burayı da birlikte inceleyelim

Reply:
Yeni commit'te çözüldü!